validphys2/src/validphys/dataplots.py

```diff
@@ -585,22 +585,23 @@ def plot_fits_chi2_spider(fits, fits_groups_chi2,
     return fig
 
 @figure
-def plot_fits_phi_spider(fits, fits_groups_data,
-                         fits_groups_data_phi, processed_metadata_group):
-   """Like plot_fits_chi2_spider but for phi."""
+def plot_fits_phi_spider(
+    fits, fits_groups_data, fits_groups_data_phi, processed_metadata_group
+):
+    """Like plot_fits_chi2_spider but for phi."""
 
-   fig = plt.figure(figsize=(12,12))
-   ax = fig.add_subplot(projection='polar')
+    fig = plt.figure(figsize=(12, 12))
+    ax = fig.add_subplot(projection='polar')
 
-   for fit, fitphi, fitgroup in zip(fits, fits_groups_data_phi, fits_groups_data):
-       phi = [exp_phi for (exp_phi, npoints) in fitphi]
-       xticks = [group.name for group in fitgroup]
+    for fit, fitphi, fitgroup in zip(fits, fits_groups_data_phi, fits_groups_data):
+        phi = [exp_phi for (exp_phi, npoints) in fitphi]
```

Comment:
I've tested this and it works fine. The tiny suggestion below just gets rid of an unused variable:
```suggestion
        phi = [exp_phi for (exp_phi, _) in fitphi]
```
and (in terms of spider plots) there is one more here:
https://github.com/NNPDF/nnpdf/blob/1e8ffd920796b59c601ac1206e6014f618235091/validphys2/src/validphys/dataplots.py#L622